gestor_absencies/serializers.py

```diff
@@ -16,6 +16,9 @@
                      VacationPolicy, Worker)
 
 
+WORKER_SENSITIVE_FIELDS = ['gender', 'category', 'contract_date']
```

Comment:
Can we parameterize this variable and put it in settings config?. I think sensitive fields can be easily change for random reasons (mostly law sh*it).
So i propose you add that list in settings, what do you think?

Reply:
You're right! We have done it in last commit